### DISCUSSION TITLE
Show organization connections in Commons when a ROR ID is included in `funderIdentifier`

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#87] [General Suggestions]
### What is the problem that your suggestion solves?

When a ROR ID is included in `funderIdentifier`, the Work is not listed as a related work for the organization with that ROR ID.

However, if a Crossref Funder ID *corresponding* to that organization's ROR ID is included, the Work is listed as a related work for that organization based on the mapping from the Crossref Funder ID to the ROR ID. See the criteria here: https://support.datacite.org/docs/connecting-to-organizations

Since the Crossref Open Funder Registry is [on track to transition to ROR](https://www.crossref.org/blog/open-funder-registry-to-transition-into-research-organization-registry-ror/), we should allow ROR IDs in `funderIdentifier` to establish connections to organizations directly.

### What solution might meet your needs?

When a ROR ID is added to a DOI in `funderIdentifier`, this should establish a connection to the organization that is visible on the Organization page.

### Your name

Kelly Stathis

### Your organization

DataCite

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by codycooperross** [comment]
In addition to Crossref Funder IDs, Organization pages now display works with a matching ROR in the work's `fundingReferences` `funderIdentifier` metadata. Take a look at [the additional works](https://commons.datacite.org/ror.org/04xeg9z08?filterQuery=fundingReferences.funderIdentifier:%22https://ror.org/04xeg9z08%22) available in Organization pages like [the National Institute of Mental Health](https://commons.datacite.org/ror.org/04xeg9z08).

We hope this change facilitates the transition to RORs during the long-term deprecation of [the Open Funder Registry](https://www.crossref.org/blog/open-funder-registry-to-transition-into-research-organization-registry-ror/). Comment with any feedback or [create a new discussion](https://github.com/datacite/datacite-suggestions/discussions/new/choose).
